src/index.js

```diff
@@ -4,14 +4,13 @@ import App from './Containers/App/App';
 import rootReducer from './reducers'
 import { BrowserRouter } from 'react-router-dom'
 import { Provider } from 'react-redux';
-import { createStore, applyMiddleware } from 'redux';
+import { createStore } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
```

Comment:
Are we planning on not using thunks?  If we are going to use them, we'll need this in there.  There's another line that applyMiddleWare(thunks) will need to be deleted if we're taking this out. Thoughts @b3an5 @kobesparrow ?

Reply:
I'm fine with adding thunks lets talk about it tomorrow morning